### DISCUSSION TITLE
Show cases / larger deployments

### DISCUSSION THREAD

**Discussion by oseiler2** [cyberman54/ESP32-Paxcounter#793] [General]
I thought it'd be nice to get some links to real world scenarios where people use pax counters in the wild beyond experiments/trials. Maybe we can get a few users to show and tell or link to their projects in this thread?

I'm also curious what realistic maximum number of people (devices) could be captured by a node. Has someone tried setting one up in a crowded environment (e.g. a big event)?

Are there any recommendations for or against certain boards for a more permanent and ideally zero maintenance deployment?

**Comment by cyberman54** [comment]
The current version in development branch is able to store up to 65535 anonymized identifiers.

![image](https://user-images.githubusercontent.com/10289836/114691232-bd429c80-9d17-11eb-8a17-af6a69dfa1ba.png)

**Reply by cyberman54** [reply]
![image](https://user-images.githubusercontent.com/10289836/114904080-c0bd4d00-9e17-11eb-9b5f-17f4dd1c44aa.png)

**Reply by oseiler2** [reply]
Nice! Is this sampling real devices using that small built-in Wifi antenna on the left?

**Reply by cyberman54** [reply]
yes and no, since new v2.5.0 version is counting universal MACs only. That means we count random MACs, not device MACs.

**Reply by oseiler2** [reply]
Any indication on how much distance a single node would cover if setup in a good location (e.g. slightly above a crowd, no obstructions)?

**Reply by cyberman54** [reply]
All depends on your antenna.

**Reply by oseiler2** [reply]
Of course... I'm just looking for some data points from other's experience here.
For example the screenshot above with the built-in antenna (looks lie a T-Beam) and ideas how large (in area/distance, not numbers) the crowd was?

**Reply by disit** [reply]
the version with anonymization is accessible on Github?

**Reply by cyberman54** [reply]
@disit Meanwhile paxcounter code was changed to ONLY process anonymous (local / random) MAC adresses, so you get anonymization with the current code.
